packages/neos-ui-redux-store/src/CR/Nodes/index.js

```diff
@@ -123,7 +144,9 @@ export const reducer = handleActions({
     [COMMENCE_REMOVAL]: contextPath => $set('cr.nodes.toBeRemoved', contextPath),
     [REMOVAL_ABORTED]: () => $set('cr.nodes.toBeRemoved', ''),
     [REMOVAL_CONFIRMED]: () => $set('cr.nodes.toBeRemoved', ''),
-    [REMOVE]: contextPath => $drop(['cr', 'nodes', 'byContextPath', contextPath])
+    [REMOVE]: contextPath => $drop(['cr', 'nodes', 'byContextPath', contextPath]),
+    [COPY]: contextPath => $set('cr.nodes.clipboard', contextPath),
+    [PASTE]: () => $set('cr.nodes.clipboard', '')
```

Comment:
The logic of keeping the copy workflow running is handled elsewhere: https://github.com/neos/neos-ui/blob/7ae1046640be35c0f780780e594ae435fc2c967c/packages/neos-ui/src/Sagas/CR/NodeOperations/index.js#L149

So, pasting multiple times is still possible